File: FactorialNumber.java
import java.util.*;
class FactorialNumber
{
    public static void main(String[]args)
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the number: ");
        int a=sc.nextInt();
        System.out.println("Factorial of "+a+" is "+fact(a));
    }
    static int fact(int n)
    {
        int result;
        if(n==0||n==1)
            return 1;
        result=fact(n-1)*n;
        return result;
    }
}

File: C/FactorialNumber.c
#include<stdio.h>
#include<conio.h>
int main()
{
    int i,x,n,f=1;
    printf("Enter a number : ");
    scanf("%d",&n);
    x = n;
    while(n>=1){
        f = f * n;
        n--;
    }
    printf("Factorial of %d is %d",x,f);
    getch();
}
